search-engine/lib/pack_factory.rb

```diff
@@ -41,18 +41,22 @@ def build_sheet(data)
     foil = false
     balanced = false
     coout = nil
+    fixed = false
 
     foil = data.delete("foil") if data.has_key?("foil")
     balanced = data.delete("balanced") if data.has_key?("balanced")
     duplicates = data.delete("duplicates") if data.has_key?("duplicates")
     count = data.delete("count") if data.has_key?("count")
+    fixed = data.delete("fixed") if data.has_key?("fixed")
 
     if balanced and duplicates
```

Comment:
We need to update the check if there are multiple flags are set, otherwise we can have weird bugs.

`[balanced, duplicates, fixes].count(&:itself) > 1` is probably the right check.

Reply:
Resolved